softwerkskammer/lib/payment/paymentService.js

```diff
@@ -5,11 +5,9 @@ const beans = require('simple-configure').get('beans');
 const PaymentInfo = beans.get('paymentInfo');
 const statusmessage = beans.get('statusmessage');
 const stripeService = beans.get('stripeService');
-const fieldHelpers = beans.get('fieldHelpers');
 
 function calcFee(amount) {
-  const fee = ((amount + 0.3) / 0.971) - amount;
-  return fieldHelpers.roundNumber(fee);
+  return ((amount + 0.3) / 0.971) - amount;
```

Comment:
das passiert jetzt mit beim formatieren. Vorher war das ziemlich schräg:
fieldHelpers exponiert die methode roundNumber, die nur hier gebraucht wird. in der Methode wird das argument formatiert. Anschließend durch index.js nochmals als Währung formatiert